tests/e2e/e2e_exec_test.go

```diff
@@ -292,6 +292,32 @@ func (s *IntegrationTestSuite) execBankSend(
 	s.executeGaiaTxCommand(ctx, c, gaiaCommand, valIdx, s.expectErrExecValidation(c, valIdx, expectErr))
 }
 
+type txBankSend struct {
+	from      string
+	to        string
+	amt       string
+	fees      string
+	log       string
+	expectErr bool
+}
+
+func (s *IntegrationTestSuite) execBankSendBatch(
+	c *chain,
+	valIdx int,
+	txs []txBankSend,
```

Comment:
```suggestion
	txs ...txBankSend,
```

Can we use a variadic function here?

Reply:
we can, but why this is better ?

Reply:
Because it can be reusable to send only one transaction without creating a slice and pass to the function. But it is more of a suggestion. Both are good